lib/simctl.js

```diff
@@ -18,42 +18,44 @@ const DEFAULT_OPTS = {
 
 /**
  * @typedef {Object} ExecOpts
- * @property {Array.<string>} args [[]] - The list of additional subcommand arguments.
+ * @property {string[]} args [[]] - The list of additional subcommand arguments.
  * It's empty by default.
  * @property {Object} env [{}] - Environment variables mapping. All these variables
  * will be passed Simulator and used in the executing function.
  * @property {boolean} logErrors [true] - Set it to _false_ to throw execution errors
  * immediately without logging any additional information.
  * @property {boolean} asynchronous [false] - Whether to execute the given command
  * 'synchronously' or 'asynchronously'. Affects the returned result of the function.
- * @property {?string} encoding - Explicitly sets streams encoding for the executed
+ * @property {string?} encoding - Explicitly sets streams encoding for the executed
  * command input and outputs.
+ * @property {string|string[]} - One or more architecture names to be enforced while
```

Comment:
I think this comment should have `architectures` after the type and before the dash.

Reply:
fixed